utils/mr/labels.py

```diff
@@ -0,0 +1,3 @@
+AUTO_MERGE = 'automerge'
```

Comment:
we'll probably need groups of labels here in some way (thinking about gitlab_housekeeping)

Reply:
Yeah.. we can extend them when the time comes